com.sap.cloud.lm.sl.cf.core/src/main/java/com/sap/cloud/lm/sl/cf/core/shutdown/model/Shutdown.java

```diff
@@ -0,0 +1,107 @@
+package com.sap.cloud.lm.sl.cf.core.shutdown.model;
+
+import java.time.ZonedDateTime;
+
+import com.google.gson.annotations.JsonAdapter;
+import com.sap.cloud.lm.sl.cf.web.api.model.ZonedDateTimeJsonAdapter;
+
+public class Shutdown {
+    private boolean isShutdowned = false;
+    @JsonAdapter(ZonedDateTimeJsonAdapter.class)
+    private ZonedDateTime shutdownRequestDate;
+    @JsonAdapter(ZonedDateTimeJsonAdapter.class)
+    private ZonedDateTime shutdownedAtDate;
+    private String appId;
+    private String appInstanceId;
+    private String appInstanceIndex;
+    private long cooldownTimeoutInSeconds;
+
+    private Shutdown(boolean isShutdowned, ZonedDateTime shutdownRequestDate, ZonedDateTime shutdownedAtDate, String appId,
+        String appInstanceId, String appInstanceIndex, long cooldownTimeoutInSeconds) {
+        this.isShutdowned = isShutdowned;
```

Comment:
It will be super if there is some kind of status which could be one of the following:
RUNNING, FINISHED, ERROR